pkg/controller/account/iam.go

```diff
@@ -141,7 +136,7 @@ func deleteAllAccessKeys(client awsclient.Client, iamUser *iam.User) error {
 
 // checkIAMUserExists checks if a given IAM user exists within an account
 // Takes a logger, an AWS client for the target account, and a target IAM username
-func checkIAMUserExists(reqLogger logr.Logger, client awsclient.Client, userName string) (bool, *iam.GetUserOutput, error) {
+func checkIAMUserExists(reqLogger logr.Logger, client awsclient.Client, userName string) (bool, *iam.GetUserOutput, error) { //nolint:deadcode
```

Comment:
Why not whack it? We can always resurrect it from the git history if we need it later.

[Later] Appears to be redundant with `CheckIAMUserExists` in pkg/awsclient/iam.go.

Reply:
I wanted to have someone else on the team evaluate whether or not to remove it.  Usually I create a card for removing a whole function just to be safe.

Thanks for the second set of eyes.  Will remove this.